src/Route.php

```diff
@@ -30,16 +30,16 @@ class Route extends Hook
     /**
      * Alias path
      *
-     * @var string
+     * @var null|string
      */
-    protected string $aliasPath = '';
+    protected ?string $aliasPath = null;
```

Comment:
One thing really weird about `Route` is it kind of maintains state via the `isAlias` flag. When a request is matched to a Route, the `Route->isAlias` flag is set to `true` and then  `App->execute()` and `App->getArguments()` knows to look at the alias for the path and params.

Since there are multiple aliases now, `Route` also needs to remember which alias path was matched. That's what I'm using this `aliasPath` for. Thoughts on this approach?